lmdeploy/api.py

```diff
@@ -76,9 +76,10 @@ def serve(model_path: str,
           chat_template_config: Optional[ChatTemplateConfig] = None,
           server_name: str = '0.0.0.0',
           server_port: int = 23333,
-          log_level: str = 'ERROR',
+          log_level: str = 'WARNING',
           api_keys: Optional[Union[List[str], str]] = None,
           ssl: bool = False,
+          block: bool = False,
```

Comment:
Is it related to chat template?

Reply:
No. Should I remove it in this PR? I can renew a PR for it.

Reply:
I think it's better to pull another PR.